app/models/custom_medication.rb

```diff
@@ -0,0 +1,4 @@
+class CustomMedication < ActiveYaml::Base
```

Comment:
I may be missing something - but why wouldn't we use our existing ProtocolDrug model along w/ some seed generation code that reads from the yml you have in this PR?

I'm sure that the med details or the 'canonical' meds will change over time, so putting them all into a static YML file feels short sighted when we have a domain model and the models in our domain for all this.

Reply:
...one follow up thought: it may be that these aren't "protocol drugs", but they are still "prescription drugs" that we can figure out how to model along side the protocol drugs. Maybe this is a subset of some new "prescription" model that we are still trying to figure out?  In any case, I think it would be worth exploring what improving our existing ActiveRecord based drug models look like rather than trying to load and update things from yaml.

Reply:
@rsanheim These are specifically non-protocol drugs that are currently entered by hand. ProtocolDrugs are not a good fit to model this.

PRD is here: https://docs.google.com/document/d/1lL43EJUxmnr7dsFjv4kyNnEyiGGbD7__4HbD_EUqpZw/edit#

Reply:
I agree that unifying our drug model so we can mark some drugs as protocol and others as custom seems like a better solution overall, but that'll likely be a significantly larger change.